src/main/java/org/openstreetmap/atlas/geography/atlas/bloated/BloatedArea.java

```diff
@@ -101,9 +119,8 @@ public int hashCode()
     @Override
     public Set<Relation> relations()
     {
-        return this.relationIdentifiers == null ? null
-                : this.relationIdentifiers.stream().map(BloatedRelation::new)
-                        .collect(Collectors.toSet());
+        return this.relationIdentifiers == null ? null : this.relationIdentifiers.stream()
```

Comment:
:( looks like we have a save action mismatch.

Reply:
I believe it's one of those cases where my save action does not agree with spotless.